base_app/services/google_service.py

```diff
@@ -24,4 +23,26 @@ def get_coordinates(self, params):
             content = {'Invalid Address': 'Please enter a street address, city, and state.'}
             return content
 
+    def get_place_id(self, params):
+        url = 'https://maps.googleapis.com/maps/api/place/findplacefromtext/json'
+        params.update({'inputtype': 'textquery'})
+        response = self.connection(url, params)
+        json = response.json()
+        if response.status_code == 200 and 'candidates' in json.keys():
```

Comment:
Do both conditions have to be met? Is it not sufficient to only check if 'candidates' is a key?

Reply:
I'm following Linda's lead here. However I could see how there might be a KeyError by trying to run `json.keys()` if `json` ended up being a null variable